data/themes/default/style.css

```diff
@@ -540,6 +540,42 @@ BBTCOView {
 	qproperty-textColor: rgb( 255, 255, 255 );
 }
 
+/* subwindows in MDI-Area */
+SubWindow {
+        color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
+                                                stop: 0 #4b525c, stop: 1.0 #31363d);
+        qproperty-activeColor: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
+                                                stop: 0 #33383e, stop: 1.0 #1a1c20);
+        qproperty-textShadowColor: rgb(0, 0, 0);
+        qproperty-borderColor: rgb(0, 0, 0);
+}
+
+/*SubWindow Title Text */
+SubWindow > QLabel {
+        color: rgb(255, 255, 255);
+        font-family: Verdana, 'Lucida Sans Unicode', sans-serif;
```

Comment:
This line should be removed. We should use the default font, as we use that for every other part of LMMS.